test/blackbox-tests/test-cases/pkg/package-command-exists.t

```diff
@@ -0,0 +1,4 @@
+The package command is an alias for the pkg command. The test ensures it breaks if
+the package command is not available.
```

Comment:
Is there really a point to this? If you remove the command, `doc/dune.inc` will already get updated.

Reply:
I didn't know if it would be enough to detect a regression. If you think this is enough, I can definitely remove this test :+1:

Reply:
It should be enough. If you remove any command, the doc file gets updated. So yeah, I think you can safely drop this test.

Reply:
I have reverted the change.